openshift-hack/e2e/annotate/rules.go

```diff
@@ -194,25 +155,13 @@ var (
 		// tests that are known flaky
 		"[Flaky]": {
 			`Job should run a job to completion when tasks sometimes fail and are not locally restarted`, // seems flaky, also may require too many resources
-			// TODO(node): test works when run alone, but not in the suite in CI
-			`\[Feature:HPA\] Horizontal pod autoscaling \(scale resource: CPU\) \[sig-autoscaling\] ReplicationController light Should scale from 1 pod to 2 pods`,
```

Comment:
I went only through a couple of tests, but for example I see this one is still present.

Or does it make sense to enable group by group in multiple PRs and with a time delay?

Reply:
It appears that this test got renamed, with the `[sig-autoscaling]` moved from the middle to the front, such that this pattern no longer matches.

So we had a rule that matched the old name of a test that used to fail, and then the test got fixed, but we still kept skipping it, and then later the test got renamed, so now the rule no longer matches, and we don't skip it, which is fine, because it passes now anyway.

Reply:
> Or does it make sense to enable group by group in multiple PRs and with a time delay?

I can change it to just print warnings rather than erroring out, and then people can do whatever they want with it... ?

Reply:
I guess, you are right. It should be fine if the tests passes anyway nowadays.

Reply:
I think it is okay to hard fail and remove the old skips. Let's see @bertinatto opinion on this.

Reply:
Let's keep it failing for now